R/prepare_ae_specific_subgroup.R

```diff
@@ -30,14 +30,17 @@
 #' @export
 #'
 #' @examples
-#' # meta <- meta_ae_example()
-#' # prepare_ae_specific_subgroup(
-#' #   meta,
-#' #   population = "apat",
-#' #   observation = "wk12",
-#' #   parameter = "rel",
-#' #   subgroup_var = "RACE"
-#' # )$data
+#' \dontrun{
```

Comment:
This example gives an error message `"Error: This function is still under development."`, so I saved it in a `\dontrun{}` environment.

Reply:
Avoid using `\dontrun` as they actually will be ran and they are likely be asked to be removed.

I see Yilong added a patch to replace `stop()` with `message()` and return `NULL` early which is the right way to go.